dev-docs/show-video-with-a-dfp-video-tag.md

```diff
@@ -65,126 +58,85 @@ Don't forget to add your own valid placement ID.
 
 ```javascript
 var videoAdUnit = {
-  code: 'video',
-  sizes: [640,480],
-  mediaType: 'video',
-  bids: [
-    {
-      bidder: 'appnexusAst',
-      params: {
-        placementId: '123456'  // <-- Replace this!
+    code: 'video',
+    sizes: [640, 480],
+    mediaType: 'video',
+    bids: [{
+        bidder: 'appnexusAst',
+        params: {
+            placementId: '9333431',
             video: {
-              skippable: true
+                skippable: true,
+                playback_methods: ['auto_play_sound_off']
             }
-      }
-    }
-  ]
+        }
+    }]
 };
 ```
 
-### 2. Implement Custom Price Buckets to work around the default CPM cap of $20
-
-By default, Prebid.js caps all CPMs at $20.  As a video seller, you may expect to see CPMs over $20.  In order to receive those bids, you'll need to implement custom price buckets using the [`setPriceGranularity`]({{site.github.url}}/dev-docs/publisher-api-reference.html#customCPMObject) method.
+### 2. Implement Custom Price Buckets
 
-For instructions, see [Custom Price Bucket with `setPriceGranularity`]({{site.github.url}}/dev-docs/examples/custom-price-bucket-using-setpricegranularity.html).
+By default, Prebid.js caps all CPMs at $20.  As a video seller, you may expect to see CPMs over $20.  In order to receive those bids, you'll need to implement custom price buckets using the [`setPriceGranularity`]({{site.baseurl}}/dev-docs/publisher-api-reference.html#customCPMObject) method.
 
-### 3. Request bids, build a video tag, and invoke the player
+For instructions, see [Custom Price Bucket with `setPriceGranularity`]({{site.baseurl}}/dev-docs/examples/custom-price-bucket-using-setpricegranularity.html).
 
-Next, do the standard Prebid "add ad units and request bids" dance.
-In the example below, we've added some code that is not strictly
-necessary, but was helpful during development.  Specifically, we log:
+### 3. Request bids, build video URL, invoke player
 
-+ the `bids` object we got back from our demand sources
-+ a notice telling us whether there was video demand
-+ the URL of the VAST creative, if any (this is also helpful if you
-  don't necessarily use DFP)
+Next, we need to do the standard Prebid "add ad units and request bids" dance.  In the example below, our callback builds the video URL the player needs using the `buildVideoUrl` method from the DFP ad server module that we built into our copy of Prebid.js in the **Prerequisites** section.
 
-To optimize setup for low latency, we recommend that this code (and that referenced above in step #1) be added to the page header.
+This method wraps the video URL in VAST tags and tries to add it to a server-side Prebid Cache. The use of the cache is transparent to the user and doesn't require a login or account of any kind.  It's designed to simplify ad ops workflows by enabling all Prebid video demand partners to serve through a single common video URL.
```

Comment:
Re: the description of how it works, thanks for the clarification.

Re: mentioning the cache, I agree with your feedback.  Gonna pull that section.  I think I was erring on the side of completeness but it's probably not appropriate for a tutorial like this.  Hopefully the average user will not have to debug any cache issues, and if they do presumably reading the code will be more helpful anyway.

Also, great idea to add the params note and link.  Adding that too.

Thanks for the helpful comments!